### PR TITLE
Use an HTTP to fetch NN details

### PR DESCRIPTION
**Pull request by Andrew-Dickinson**
Restructure DB class based on HTTP calls. Also clean up a few environment variable messes

Keeps the existing client around under a new name, and establishes a shared interface for backwards compatibility

### CODE REVIEW

File: supportbot/supportbot/app.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def run_app(config):
 
     app = App(token=os.environ.get("SLACK_BOT_TOKEN"))
 
-    database_client_cached = DatabaseClient(os.environ.get("SPREADSHEET_ID"))
+    database_client_cached = MeshDBDatabaseClient(os.environ.get("MESHDB_AUTH_TOKEN"))
```
Comment (andybaumgar) [inline supportbot/supportbot/app.py]: Is this actually cached?  The old client pulled all data on initialization.  Maybe rename otherwise?

**Review by andybaumgar — APPROVED** [review]
Looks good.  Added one nit.

File: supportbot/supportbot/app.py (hunk 1 above)
Comment (Andrew-Dickinson) [reply supportbot/supportbot/app.py]: Not cached. It pulls at request time. Renamed

**Comment by Andrew-Dickinson** [comment]
Whoops I didn't mean to close this and GH won't let me reopen it. It needs to be updated probably anyway so I guess I'll leave it closed for now

**Pull request closed without merging**